### ISSUE TITLE
Finnish / Suomi translation (attached file)

### ISSUE THREAD

**Issue by huuhaa** [WangDaYeeeeee/GeometricWeather#294]
I decided to contribute to Finnish / Suomi translation, but sending translation now as s zip, not push request cause..

1) I'm too used to online translation tools like Weblate etc. so not 100% sure should I just fork values and at my owb fork rename folder as values_fi and add translations and then push request, or what is proper way..

2) I'm also not 100% sure did I translate some at arrays.xml that I shouldn't. Tied to follow those quides though..

3) Is values_fi folder itself enough, I don't remember seeing anywhere at translations ehere to put language which this translation is for, except that folder.

Anyway I hope this translation is done in correct way and you could add Finnish translation to this awesoe app. In future it should be easier to contribute via using github since those files are at their place. Sure I wouldn't mind if at some point you will offer also option to translate via weblate.

Translation here at attachment:

[values_fi.zip](https://github.com/WangDaYeeeeee/GeometricWeather/files/8274739/values_fi.zip)

**Comment by papjul** [comment]
1. If you are not technical, just use the Edit button from GitHub, this will automatically create a fork and pull request.

2. Have a look at values-fr which is up-to-date. Array values not in values-fr/arrays.xml are technical and should not be added to your values-fi/arrays.xml.

It's also written on top of this file:
```
<!-- Do not translate those arrays which named "xxx_values". -->
<!-- Do not translate those items with a value of "@string/xxx". -->
```

3. Yes, just arrays and strings from values folder is enough. However, you can optionally translate the fastlane directory if you want translations in F-Droid of the following items:
- Description
- Changelog
- Screenshots (take the same screenshots with your language set)

**Comment by huuhaa** [comment]
Ok, I will take a look from values-fr so if I push edit there, I can change it also to vaölues-fi before pull request?

I did notice that written at top of file, but was bit unsure still & tried to look bit from german file as an example, but that was totally different looking than default eglish one is, if I remember correct.

I will wait your reply before starting to edit via GitHub.

**Comment by papjul** [comment]
It seems the directory values-fi doesn't exist yet, are you asking how you can create it from GitHub?

I have no idea as I am not using this website/app but it seems at least you have a "Add file" on top right of the directory res/ on the website so maybe you can create a directory as well?

**Comment by huuhaa** [comment]
@papjul  Yes I meant how to create folder with web UI, it is with add file, specifying /folder/filename.xml

I finally did those changes and tried to contribute that way & leaved pull reguest. Unfurtunately at first there was stupid typo strinds.xml I went back to step where I could edit it as strings.xml but it might have left pull reguest at totally 3 files for that reason.

I didn't edit strings.xml anymore since I understood previously it was ok. Used same as then and for arrays.xml I used French file as basefile. I hope everything is ok by now.
